### PR TITLE
Tejus

### PR DESCRIPTION
**Pull request by tejuscs**
# Add here your WMS ID
11714

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [Y ] All filenames are lower case (including folders, images, files, etc.)
- [ Y] Filenames are descriptive
- [ Y] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ Y] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ Y] Image references in markdown contain an alternate text

### CODE REVIEW

File: adb-azure/workshop/db-users/index.html
Comment (klazarz) [inline adb-azure/workshop/db-users/index.html:1]: please change https://developer.oracle.com/livelabs to https://livelabs.oracle.com/

**Review by klazarz — COMMENTED** [review]
@tejuscs please see my comment about index.html

File: adb-azure/workshop/db-users/index.html
Comment (tejuscs) [reply adb-azure/workshop/db-users/index.html:1]: Changed.

**Comment by klazarz** [comment]
@tejuscs can you please resolve the conflicts?

**Pull request closed without merging**